### PR TITLE
meta-iotqa: add qa tests for opencv python interface

### PR DESCRIPTION
**Pull request by epyeoh**
Add tests for opencv python interface.

Signed-off-by: Yeoh Ee Peng <ee.peng.yeoh@intel.com>

### CODE REVIEW

**Comment by epyeoh** [comment]
test this please

**Comment by epyeoh** [comment]
test this please

**Comment by epyeoh** [comment]
test this please

**Comment by okartau** [comment]
This fails in CI in way which is not easily detectable, xUnit results processing marks job status FAILURE (search for FAILURE in job log), but it's not mentioned that test cases were failed, also no failures visible in xUnit report summary.
I guess tehre is something wrong in how some new test results are presented/formatted

**Comment by okartau** [comment]
root cause is not mentioned in top level log and that is bit less user-friendly,
but I found the real error pointed out in one of on-device log files
(see test_aft.570x.refkit-image-computervision.log in archived files for job 2280):

Traceback (most recent call last):
  File "iottest/runtest.py", line 256, in <module>
    ret = main()
  File "iottest/runtest.py", line 250, in main
    runner.loadTests()
  File "iottest/runtest.py", line 103, in loadTests
    super(RuntestTestContext, self).loadTests()
  File "/root/workspace/workspace/intel-iot-refkit_pull-requests_5/iottest/oeqa/oetest.py", line 298, in loadTests
    suites = [testloader.loadTestsFromName(name) for name in self.testslist]
  File "/root/workspace/workspace/intel-iot-refkit_pull-requests_5/iottest/oeqa/oetest.py", line 298, in <listcomp>
    suites = [testloader.loadTestsFromName(name) for name in self.testslist]
  File "/usr/lib/python3.4/unittest/loader.py", line 105, in loadTestsFromName
    module = __import__('.'.join(parts_copy))
  File "/root/workspace/workspace/intel-iot-refkit_pull-requests_5/iottest/oeqa/runtime/multimedia/opencv/opencv_cv2.py", line 36
    self.target.copy_to(
                       ^
TabError: inconsistent use of tabs and spaces in indentation

**Comment by epyeoh** [comment]
Olev, thank you so much for helping to debug the issue, it help a lot!

**Comment by epyeoh** [comment]
tested this please

**Comment by okartau** [comment]
test this please

**Comment by okartau** [comment]
Test this please
(was wifi related failure on tester)

**Comment by epyeoh** [comment]
@pohly hi Patrick, will it be ok to remove the [test-build] tag as this pull request is ready for working as expected and ready for review?

File: meta-iotqa/lib/oeqa/runtime/multimedia/opencv/opencv_cv2.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    def test_cv_detect_shape(self):
+        (status, output) = self.target.run('python %s' % OpenCVTest.cv_detect_shape_target)
+        self.assertEqual(status, 0, msg="Error messages: %s" % output)
+		
+    def tearDown(self):
+        '''
+        Clean work: remove all the files copied to the target device.
+        @fn tearDown
+        @param self
+        @return
```
Comment (pohly) [inline meta-iotqa/lib/oeqa/runtime/multimedia/opencv/opencv_cv2.py]: You diligently added annotations for the function prototype, but without any additional information.

Why are such annotations useful? If they are not useful, please remove them.

A general comment about the tests: please explain somewhere (head of file?) what the tests do in general and (in each test) what the specific test does.

Comment (epyeoh) [inline meta-iotqa/lib/oeqa/runtime/multimedia/opencv/opencv_cv2.py]: Agreed. I will improve the comment and resubmit again. Thank you for your advice!

**Pull request merged**